meltano.yml

```
@@ -19,7 +19,7 @@ plugins:
         keys:
         - col1
     settings:
-      - description: Array of objects containing keys: `entity`, `file`, `keys`,  and `encoding` (Optional)
+      - description: Array of objects containing keys: `entity`, `file`, `keys`, `encoding` (Optional), `delimiter` (Optional), `doublequote` (Optional), `escapechar` (Optional), `quotechar` (Optional), `skipinitialspace` (Optional), `strict` (Optional)
```

```suggestion
      - description: Array of objects containing keys - `entity`, `file`, `keys`, `encoding` (Optional), `delimiter` (Optional), `doublequote` (Optional), `escapechar` (Optional), `quotechar` (Optional), `skipinitialspace` (Optional), `strict` (Optional)
```

@Bartman0 this wasnt a change you made but I realized while reviewing that the colon breaks the yaml syntax. Your updates here look great.